core/cas-server-core-authentication-api/src/main/java/org/apereo/cas/authentication/principal/resolvers/ChainingPrincipalResolver.java

```diff
@@ -84,17 +83,7 @@ public Principal resolve(final Credential credential, final Optional<Principal>
                 }
             }
         });
-        final int count = principals
-            .stream()
-            .map(p -> p.getId().trim().toLowerCase())
-            .collect(Collectors.toSet())
-            .size();
-        if (count > 1) {
-            throw new PrincipalException("Resolved principals by the chain are not unique because principal resolvers have produced CAS principals "
```

Comment:
Please restore this block and perhaps instead of throwing an error, log a message to indicate the behavior introduced below.